services/src/test/scala/cromwell/pipeline/service/UserServiceTest.scala

```diff
@@ -49,32 +55,40 @@ class UserServiceTest extends AsyncWordSpec with Matchers with MockitoSugar {
     }
 
     "updateUser" should {
-      "returns success if database handles query" in {
-        val userId = "123"
-        val user = User(UserId("123"), "email@cromwell.com", "hash", "salt", "name", "lastName")
-        val user2 = user.copy(email = "updatedEmail", firstName = "updatedFirstName", lastName = "updatedLastName")
+      "returns success if database handles query" taggedAs (Service) in {
+        val numberId = "123"
+        val updatedUser =
+          dummyUser.copy(email = "updatedEmail", firstName = "updatedFirstName", lastName = "updatedLastName")
         val request = UserUpdateRequest("updatedEmail", "updatedFirstName", "updatedLastName")
 
-        when(userRepository.getUserById(UserId("123"))).thenReturn(Future(Some(user)))
-        when(userRepository.updateUser(user2)).thenReturn(Future.successful(1))
+        when(userRepository.getUserById(userId)).thenReturn(Future(Some(dummyUser)))
+        when(userRepository.updateUser(updatedUser)).thenReturn(Future.successful(1))
 
-        userService.updateUser(userId, request).map { result =>
+        userService.updateUser(numberId, request).map { result =>
           result shouldBe 1
         }
       }
     }
 
     "updatePassword" should {
-      "returns success if database handles query" in {
+      "returns success if database handles query" taggedAs (Service) in {
         val id = "123"
         val salt = "salt"
         val user =
-          User(UserId(id), "email@cromwell.com", calculatePasswordHash("password", salt), salt, "name", "lastName")
+          User(
+            UserId(id),
+            "email@cromwell.com",
+            StringUtils.calculatePasswordHash("password", salt),
+            salt,
+            "name",
+            "lastName"
+          )
```

Comment:
Make it one line statement or make arguments named

Reply:
Done